### ISSUE TITLE
Issue when starting AACAdditionPro

### ISSUE THREAD

**Issue by duplexsystem** [Photon-GitHub/AACAdditionPro#396]
```[17:17:17 ERROR]: Error occurred while enabling AACAdditionPro v6.2.1 (Is it up to date?)
java.lang.IncompatibleClassChangeError: Inconsistent constant pool data in classfile for class de/photon/aacadditionpro/modules/Dependency. Method 'boolean lambda$allowedToStart$0(java.lang.String)' at index 36 is CONSTANT_MethodRef and should be CONSTANT_InterfaceMethodRef
at de.photon.aacadditionpro.modules.Dependency.allowedToStart(Dependency.java:17) ~[?:?]
at de.photon.aacadditionpro.modules.Module.enableModule(Module.java:24) ~[?:?]
at de.photon.aacadditionpro.modules.ModuleManager.<init>(ModuleManager.java:25) ~[?:?]
at de.photon.aacadditionpro.AACAdditionPro.onEnable(AACAdditionPro.java:153) ~[?:?]
at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.15.2.jar:git-Tuinity-"037f5ca"]
at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:364) ~[patched_1.15.2.jar:git-Tuinity-"037f5ca"]
at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.15.2.jar:git-Tuinity-"037f5ca"]
at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:470) ~[patched_1.15.2.jar:git-Tuinity-"037f5ca"]
at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:384) ~[patched_1.15.2.jar:git-Tuinity-"037f5ca"]
at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:488) ~[patched_1.15.2.jar:git-Tuinity-"037f5ca"]
at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:299) ~[patched_1.15.2.jar:git-Tuinity-"037f5ca"]
at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:895) ~[patched_1.15.2.jar:git-Tuinity-"037f5ca"]
at java.lang.Thread.run(Thread.java:834) [?:?]```

**Comment by Photon-GitHub** [comment]
There are a few things that could cause the error, so here are a few directions:
1) Update Spigot / Paper
2) Redownload AACAdditionPro 
3) Make sure you have only one AACAdditionPro in your plugins folder
4) Restart your server in case you tried reloading.

Generally please report errors with more information, such as your Spigot / Paper version.

**Comment by budgidiere** [comment]
1. I am using latest Paper build.
2. Tried that. Same error.
3. Only one.
4. Did not try reloading.
I also have latest AAC version.

**Comment by Photon-GitHub** [comment]
Might be fixed in 6.3.0

**Comment by Photon-GitHub** [comment]
Closed due to inactivity. Please comment and reopen if the issue persists.
